openforcefield/utils/toolkits.py

```diff
@@ -268,62 +270,62 @@ class OpenEyeToolkitWrapper(ToolkitWrapper):
         'SLN', 'SMI', 'USM', 'XYC'
     ]
 
+    # TODO: AR - Do we need both toolkit_is_available() and is_available()?
```

Comment:
I don't think so. We could remove `is_available()` now, or if it would take a while to track down and replace all the usages, we could make an issue to clean it up.

Reply:
I agree. I'd remove `toolkit_is_available` however. The user would already use it like
```
toolkit = RDKitToolkitWrapper()
toolkit.is_available()
```
instead of `toolkit.toolkit_is_available()`. I'll open a separate issue anyway.

Reply:
cc #214